tests/unit/commands/samconfig/test_samconfig.py

```diff
@@ -1027,6 +1203,97 @@ def test_sync(
                 {"HelloWorld": ["file.txt", "other.txt"], "HelloMars": ["single.file"]},
             )
 
+    @patch("samcli.commands._utils.experimental.is_experimental_enabled")
+    @patch("samcli.lib.cli_validation.image_repository_validation._is_all_image_funcs_provided")
+    @patch("samcli.lib.cli_validation.image_repository_validation.get_template_artifacts_format")
+    @patch("samcli.commands._utils.template.get_template_artifacts_format")
+    @patch("samcli.commands._utils.options.get_template_artifacts_format")
+    @patch("samcli.commands.sync.command.do_cli")
+    def test_sync_with_no_use_container(
```

Comment:
qq: is this test suppose to set the `--no-use-container` option? Otherwise at first glance, it looks like the test is checking the same behaviour as before (ie. this uses the config value and we didn't specify any CLI options)